romana-install/roles/stack/devstack/prep/tasks/disable_virbr0.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
The filename implies that the virbr0 interface is removed. The content of the file, however, talks about libvirt. Maybe add a comment to explain this.